pybind_instructions.md

```diff
@@ -0,0 +1,82 @@
+# Porting an algorithm from C++ to Python
+
+## Table of Contents
+- [Why?](#why)
+- [How it works](#how-it-works)
+- [Requirements](#requirements)
+
+## Why?
+The goal of this project is to allow users to take an algorithm written in C++ and translate it into runnable Python code.
+This could allow a user to develop C++ code and run it on the EmotiBit's firmware, software, and through Python, hitting three different metrics.
+
+## How it works
```

Comment:
This looks like its out of  https://github.com/EmotiBit/EmotiBit_Plugins/blob/main/README_py.md#how-it-works.
Let's try to maintain single source of truth. Maybe pointing to the PLugins repo is the right path forward? We can discuss it.